app/views/publishers/vacancies/_tabs_show.html.slim

```diff
@@ -2,6 +2,7 @@
   - tabs.navigation_item text: t("buttons.manage_listing"), link: organisation_job_path(vacancy.id), active: controller_name == "vacancies"
   - if vacancy.can_receive_job_applications? && !current_organisation.local_authority?
     - tabs.navigation_item text: t("tabs.applications.hide_count"), link: organisation_job_job_applications_path(vacancy.id), active: controller_name == "job_applications"
-  - tabs.navigation_item text: t("tabs.statistics"), link: organisation_job_statistics_path(vacancy.id), active: controller_name == "statistics"
+  - if vacancy.listed?
```

Comment:
`vacancy.listed?` will discard expired vacancies, for which you may still want to see the statistics maybe? if that's the case you want to use `vacancy.published?`